src/components/cards/CardsContainer.svelte

```diff
@@ -23,6 +23,9 @@
   }
 </style>
 
-<ul style="--gap:{gridGap}rem; --card-width:{cardWidth}rem">
+<ul
+  class="card--list--container"
```

Comment:
added a class here to target it in `_carbon-overrides.scss`

Reply:
@mukhtyar could we make the class name more similar to the Component name? Maybe `cards-container-list`? That way it's a little easier to find what it relates to in the Carbon style overrides `.scss` partial.

Reply:
updated class name in https://github.com/berkeley-gif/caladapt-website-2021/pull/148/commits/a9a6094666d89e45b8e526a6843f71572bb55959